active/operators/bort/generate_job_id.py

```diff
@@ -26,16 +26,18 @@ def generate_job_id(**kwargs):
     job_start_date = datetime.now().strftime(common_variables.DT_FORMAT)[:-3]
     bortdates = []
 
-    blob_job_queue = _get_blobs_since_date(date=last, job_state=common_variables.JOB_COMPLETE)
+    blob_job_queue = _get_blobs_since_date(date=last_bort_update_date, job_state=NLP)
```

Comment:
is NLP a common_variable?

Reply:
should it be common_variables.NLP_NER_COMPLETE ?

Reply:
yeah, this should be common_variables.NLP_NER_COMPLETE